test/api/accounts.js

```diff
@@ -98,6 +99,14 @@ describe('Accounts API', () => {
 			});
 		});
 
+		it('using known lowercase address should be ok', (done) => {
```

Comment:
This test doesn't test your implementation. You should test `getTransactionsByAddress` rather than `getAccount`